pkg/printers/testing/common.go

```diff
@@ -0,0 +1,205 @@
+// Copyright 2022 The Kubernetes Authors.
```

Comment:
pkg/printers/testing should probably be pkg/printers/testutil to match the other few testutil packages